resources/js/export.js

```diff
@@ -0,0 +1,410 @@
+import * as dayjs from "dayjs";
+import * as dayjsQuarterOfYear from "dayjs/plugin/quarterOfYear";
+import * as dayjsLocalizedFormat from "dayjs/plugin/localizedFormat";
+
+import { TransactionsApi } from "./transactions-api";
+
+dayjs.extend(dayjsQuarterOfYear);
+dayjs.extend(dayjsLocalizedFormat);
+
+const TransactionsExport = ({
+    address,
+    userCurrency,
+    rate,
+    network,
+    canBeExchanged,
+}) => {
+    const csvColumns = {
+        id: "TXID",
+        timestamp: "Timestamp",
+        sender: "Sender",
+        recipient: "Recipient",
+        amount: "Value [:networkCurrency]",
+        fee: "Fee [:networkCurrency]",
+        total: "Total [:networkCurrency]",
+        amountFiat: "Value [:userCurrency]",
+        feeFiat: "Fee [:userCurrency]",
+        totalFiat: "Total [:userCurrency]",
+        rate: "Rate [:userCurrency]",
+    };
+
+    const getTransactionAmount = (transaction) => {
+        let amount = transaction.amount / 1e8;
+        if (transaction.type === 6 && transaction.typeGroup === 1) {
+            return transaction.asset.payments.reduce(
+                (totalAmount, recipientData) => {
+                    if (recipientData.recipientId === address) {
+                        if (totalAmount < 0) {
+                            totalAmount = 0;
+                        }
+
+                        totalAmount += recipientData.amount / 1e8;
```

Comment:
change this to a method that takes arktoshi and changes it to a "regular" number, as it's reused quite a bit